apps/paymaster-proxy/src/db/Database.ts

```diff
@@ -0,0 +1,41 @@
+import type { NodePgDatabase } from 'drizzle-orm/node-postgres'
+import { drizzle } from 'drizzle-orm/node-postgres'
+import { migrate } from 'drizzle-orm/node-postgres/migrator'
+import { Pool } from 'pg'
+
+export type Database = NodePgDatabase
+
+export type DatabaseOptions = {
+  user: string | undefined
+  database: string | undefined
+  host: string | undefined
+  port: number | undefined
+  password?: string | undefined
+  max?: number
+}
+
+export type MigrationOptions = {
+  migrationsFolder?: string | undefined
+}
+
+const DEFAULT_DATABASE_MIGRATION_FOLDER = 'migrations'
+
+export function connectToDatabase(options: DatabaseOptions): Database {
+  const pool = new Pool(options)
+  return drizzle(pool)
+}
+
+/**
+ * This is a temporary function until we get our utility pod setup to run migrations
+ */
+export async function runMigrations(
```

Comment:
can be in a separate pr but dont forget to add a call to this in the service initialization! you will also want to wrap it in a retry + backoff since there is a race condition between the service starting up and the sql server being available to connect to